Update button docs, add info for controls samples on the store and up… (#911)
telerik/maui-docs: controls/collectionview/drag-and-drop/overview.md

# .NET MAUI CollectionView Drag and Drop

The .NET MAUI CollectionView provides drag and drop functionality which allows the end user to reorder its items.

![.NET MAUI CollectionView Reorder Items](../images/collectionview-itemsreorder.gif)

> To enable the reorder functionality, set the `IsDragDropEnabled` property to `True`. To start reordering the items, the end user has to click and hold an item on mobile (Android and iOS) and on dekstop (WinUi and MacCatalyst) after clicking on the item, the item can be dragged. In this way, the drag is triggered and the user can move and release (drop) the item at the desired position. This also performs a reorder operation on the data. 

Reviewer: ```suggestion
To enable the reorder functionality, set the `IsDragDropEnabled` property to `True`. To start reordering the items, the end user must tap and hold an item on mobile (Android and iOS), while on dekstop (WinUi and MacCatalyst), after clicking the item, the item can be dragged. In this way, the drag is triggered and the user can move and release (drop) the item at the desired position. This also performs a reorder operation on the data. 
```
